### DISCUSSION TITLE
Use folder names for osu!lazer skin dropdown.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#21997] [Ideas] [answered]
In my opinion it is very hard for me to find my skins in lazer because it uses the skin.ini file instead of the folder name. Due to lots of people coming from stable this might confuse them.

**Comment by peppy** [comment, marked as answer]
Folder names are already appended to the end for now, because some people actually use the internal names of skins.

Eventually you'll be able to rename the skins in-game so you can adjust them to your liking.
